src/main/java/org/primefaces/component/chips/ChipsRenderer.java

```diff
@@ -143,8 +144,13 @@ protected void encodeMarkup(FacesContext context, Chips chips) throws IOExceptio
         writer.writeAttribute("class", "ui-widget", null);
         writer.writeAttribute("name", inputId, null);
         writer.writeAttribute("autocomplete", "off", null);
+        if (disabled) {
+            writer.writeAttribute("disabled", "disabled", "disabled");
+        }
+        if (chips.isReadonly()) {
+            writer.writeAttribute("readonly", "readonly", "readonly");
+        }
 
```

Comment:
I put these back because the accessibility attributes were moved to the UL instead of each LI